packages/fxa-settings/src/images/index.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { ReactComponent as HeartsBroken } from './graphic_hearts_broken.svg';
+import { ReactComponent as HeartsVerified } from './graphic_hearts_verified.svg';
+import { ReactComponent as RecoveryCodes } from './graphic_recovery_codes.svg';
+import { ReactComponent as TwoFactorAuth } from './graphic_two_factor_auth.svg';
+import { ReactComponent as Mail } from './graphic_mail.svg';
+
+import { useFtlMsgResolver } from '../models';
+
+// Use this component to add your image into the collection of images exported below.
+// If the component is not being reused across directories, it should remain inline
```

Comment:
~inline~ `in the component or page directory where it's used` maybe? Or do you literally mean like an inlined SVG?

Reply:
Oh nice yes that's clearer!